python/ray/air/_internal/mlflow.py

```diff
@@ -14,6 +14,25 @@
 logger = logging.getLogger(__name__)
 
 
+DATABRICKS_HOST = "DATABRICKS_HOST"
+DATABRICKS_TOKEN = "DATABRICKS_TOKEN"
+DATABRICKS_CLIENT_ID = "DATABRICKS_CLIENT_ID"
+DATABRICKS_CLIENT_SECRET = "DATABRICKS_CLIENT_SECRET"
+
+
+def is_in_databricks_runtime():
+    return "DATABRICKS_RUNTIME_VERSION" in os.environ
+
+
+def verify_databricks_auth_env():
```

Comment:
Is there a way for us to short-circuit anything here if the databricks-sdk is installed on the Ray cluster? Is the workspace API able to fetch credentials if it is installed?

Reply:
No.

Databricks sdk will try to read credentials by the following ways:
* read environmental variables , **But user can't easily set environmental variables in spark worker side, except putting the environmental variables in spark cluster config**
* read ~/.databrickscfg file (not available)
* read dynamic token by `dbutils`, **this is not suitable for Ray cluster, because the dynamic token will be refreshed periodically**

so, letting user to manually setting credential environmental variables should be the best way. Other automatic way is not reliable and complicate issues.

Reply:
Sounds good! Since this is a familiar operation anyway with other uses of MLflow, it seems fine :)